pkg/kapis/openpitrix/v1/register.go

```diff
@@ -36,17 +36,14 @@ const (
 var GroupVersion = schema.GroupVersion{Group: GroupName, Version: "v1"}
 
 func AddToContainer(c *restful.Container, factory informers.InformerFactory, op op.Client) error {
-	if op == nil {
```

Comment:
why do we remove this

Reply:
In tools/cmd/doc-gen/main.go:118
```go
	urlruntime.Must(openpitrixv1.AddToContainer(container, informerFactory, nil))
```
After deletion, this register will be consistent with other register behavior.